gobblin-modules/gobblin-azure/src/main/java/org/apache/gobblin/azure/adf/ADFConfKeys.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.gobblin.azure.adf;
+
+public class ADFConfKeys {
```

Comment:
`ConfigurationKeys` type of class is not a good idea ( although we've done things like that a lot), because:
- 1) Configuration keys should stay in the class that being most relevant semantically. 
- 2) Having a lots of public-static-final is a problem since any constructs and access these, if they shared the common state object ( which is common pattern that gobblin does ), we lost the control of access-modifiers, meaning a property can be changed anywhere. This is a historical problem but I would say we should not make the problem worse.

Reply:
will change